com.microsoft.mrtk.input/Interactors/InteractorVisuals/Lines/LineUtility.cs

```diff
@@ -217,5 +218,87 @@ public static Vector3 GetEllipsePoint(Vector2 radius, float angle)
         /// Used to calculate the ellipse point in <see cref="GetEllipsePoint"/>
         /// </summary>
         private static Vector3 cachedEllipsePoint = Vector3.zero;
+
+        /// <summary>
+        /// Used to locate and lock the raycast hit data on a select.
+        /// </summary>
+        /// <param name="rayInteractor">The XRRayInteractor responsible for the raycast hit.</param>
+        /// <param name="interactableObject"> The IXRSelectInteractable which has been selected. </param>
+        /// <returns>The local position and normal of the hit target, the hit target transform, and a reference point to calculate hit distance, contained in a TargetHitInfo struct.</returns>
+        public static TargetHitInfo LocateTargetHitPoint(this XRRayInteractor rayInteractor, IXRSelectInteractable interactableObject)
```

Comment:
I'm not sure "hit target" stuff makes sense in Line utilities.   Line utilities seems more related to rendering lines, not interactors hits.  I think this function can be useful for other things not related to line visuals or reticles.

Maybe create a new class under the `Interactors` directory and call it `IXRInteractorExtensions`